Paymetheus/ViewModels/PurchaseTicketsViewModel.cs

```diff
@@ -0,0 +1,403 @@
+﻿using Paymetheus.Decred;
+using Paymetheus.Decred.Util;
+using Paymetheus.Decred.Wallet;
+using Paymetheus.Framework;
+using Grpc.Core;
+using System;
+using System.Windows;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace Paymetheus.ViewModels
+{
+    class PurchaseTicketsViewModel : ViewModelBase
+    {
+        public PurchaseTicketsViewModel() : base()
+        {
+            var synchronizer = ViewModelLocator.SynchronizerViewModel as SynchronizerViewModel;
+            if (synchronizer != null)
+            {
+                SelectedAccount = synchronizer.Accounts[0];
+            }
+
+            FetchStakeDifficultyCommand = new DelegateCommand(FetchStakeDifficultyAsync);
+            FetchStakeDifficultyCommand.Execute(null);
+
+            _purchaseTickets = new DelegateCommand(PurchaseTicketsAction);
+            _purchaseTickets.Executable = false;
+        }
+
+        private DelegateCommand _purchaseTickets;
+        public ICommand Execute => _purchaseTickets;
+
+        private bool _poolChecked = false;
+        public bool PoolChecked
+        {
+            get { return _poolChecked; }
+            set { _poolChecked = value; RaisePropertyChanged(); }
+        }
+
+        private bool _feesChecked = false;
+        public bool FeesChecked
+        {
+            get { return _feesChecked; }
+            set { _feesChecked = value; RaisePropertyChanged(); }
+        }
+
+        private AccountViewModel _selectedAccount;
+        public AccountViewModel SelectedAccount
+        {
+            get { return _selectedAccount; }
+            set { _selectedAccount = value; RaisePropertyChanged(); }
+        }
+
+        private Address _ticketAddress = null;
+        private string _ticketAddressString = "";
+        public string TicketAddressString
+        {
+            get { return _ticketAddress.ToString(); }
```

Comment:
this throws a NRE when _ticketAddress is null.  Use `?.`:

`_ticketAddress?.ToString() ?? ""`

?. propagates the null and if null, defaults to the empty string instead.